docs/survey.md

```diff
@@ -40,128 +44,246 @@
-- ( ) Mostly as part of my job (employee)
-- ( ) Mostly as part of my job (self-employed)
+- ( ) Mostly during my work time
 
-### ...I feel I can sustain the project for a long time
+### ...how difficult is it for you to maintain your project over a long time?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
 
-### ...I feel I get sufficient help from the community
+### ...how difficult is it for you to get help from the community?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
+
+### ...how much would money help the maintainance of your project?
+
+- ( ) It would help a lot
+- ( ) It would help somewhat
+- ( ) I'm not sure
+- ( ) It wouldn't help much
+- ( ) It wouldn't help at all
+
+### ...how easy to you feel it is to choose a license that helps you realize your open source goals?
+
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
+
+### ...how happy are you with the official documentation on building and maintaining open source projects?
+
+**Filter: Next question is only visible if the answer is 4-5**
+
+- ( ) Very happy
+- ( ) Somewhat happy
+- ( ) Neither happy nor unhappy
+- ( ) Somewhat unhappy
+- ( ) Very unhappy
+
+## What is the reason you aren't happy with the guidance?
+
+* [-----------]
 
 ## As an open source contributor...
 
-### ...I am doing my work:
+### ...where do you do most of your work?
 
 - ( ) Mostly in my spare time
 - ( ) A combination of spare time and work time
-- ( ) Mostly as part of my job (employee)
-- ( ) Mostly as part of my job (self-employed)
+- ( ) Mostly during my work time
 
-### ...I feel contributing to projects is usually quite easy
+### ...how difficult do you find contributing to projects?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
 
-### ...I generally feel maintainers value my contributions
+### ...how valued do you feel your contributions are by the maintainers?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very valued
+- ( ) Somewhat valued
+- ( ) Neither valued nor unvalued
+- ( ) Somewhat unvalued
+- ( ) Very unvalued
 
 ## As an open source consumer...
 
-### ...I'm generally not concerned adding dependencies on libraries not provided by myself
+### ...how confident are you with taking a dependency on a new open source project?
```

Comment:
Would it make sense to see if people require a # of downloads/stars/contributes before trusting an OSS project and if so, what those metrics are?

Reply:
And should we ask what licenses are typically allowed by companies?

Reply:
Both are interesting but I think we'd ask those question during the interviews. The survey is more designed to identify who they are (maintainer, contributor, consumer) and how they think/feel about the .NET ecosystem.